bundles/org.openhab.binding.opensprinkler/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -60,6 +60,10 @@
 			<channel id="ignoreRain" typeId="ignoreRain"></channel>
 		</channels>
 
+		<properties>
+			<property name="thingTypeVersion">1</property>
+		</properties>
```

Comment:
Why did you add `thingTypeVersion` property to the station thing type while this thing type is not updated ?

Reply:
No reason, so I have removed your suggestion.